catalog/message/schema/catalog-schema.json

```diff
@@ -1,39 +1,381 @@
+                    }
+                },
+                "dct:identifier": {
+                    "type": "string"
+                },
+                "dct:isReferencedBy": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dct:issued": {
+                    "$ref": "#/definitions/time"
+                },
+                "dct:language": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dct:license": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dct:modified": {
+                    "$ref": "#/definitions/time"
+                },
+                "dct:publisher": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dct:relation": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "dct:title": {
+                    "type": "string"
+                },
+                "dct:type": {
+                    "$ref": "#/definitions/Reference"
+                },
+                "odrl:hasPolicy": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "https://raw.githubusercontent.com/International-Data-Spaces-Association/ids-specification/main/negotiation/message/schema/contract-schema.json#/definitions/Policy"
+                    },
+                    "minItems": 1
+                }
+            }
+        },
+        "Dataset": {
+            "type": "object",
+            "allOf": [
+                {
+                    "$ref": "#/definitions/Resource"
+                },
+                {
+                    "properties": {
+                        "dcat:distribution": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/Distribution"
+                            },
+                            "minItems": 1
+                        },
+                        "dcat:spatialResolutionInMeters": {
+                            "$ref": "#/definitions/Reference"
+                        },
+                        "dcat:temporalResolution": {
+                            "$ref": "#/definitions/duration"
+                        },
+                        "dct:accrualPeriodicity": {
+                            "$ref": "#/definitions/Reference"
+                        },
+                        "dct:spatial": {
+                            "$ref": "#/definitions/Reference"
+                        },
+                        "dct:temporal": {
+                            "$ref": "#/definitions/Reference"
+                        },
+                        "prov:wasGeneratedBy": {
+                            "$ref": "#/definitions/Reference"
                         }
+                    }
+                }
+            ]
+        },
+        "DataService": {
```

Comment:
DataService should also include Resource as it is a subclass of Resource in DCAT